README.md

```diff
@@ -29,7 +29,7 @@ Role Variables
 Dependencies
 ------------
 
-None
+`ansible-galaxy collection install community.network`
```

Comment:
How about using a [requirements.yaml file](https://galaxy.ansible.com/docs/using/installing.html#installing-multiple-roles-from-a-file) and typing this in the README:

```suggestion
Install using:

- `ansible-galaxy -r requirements.yaml install`
```

Reply:
This is a better documentation of using collection in `requirements.yaml`: https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#install-multiple-collections-with-a-requirements-file